sql-database/transparent-data-encryption/setup-tde-byok-sqlmi.ps1

```diff
@@ -4,23 +4,21 @@ Connect-AzAccount
 # If there are multiple subscriptions, choose the one where AKV is created: 
 Set-AzContext -SubscriptionId "subscription ID"
 
-# Install the preview version of AzureRM.Sql PowerShell package 4.11.6-preview if you are running this PowerShell locally (uncomment below):
-# Install-Module -Name AzureRM.Sql -RequiredVersion 4.11.6-preview -AllowPrerelease
+# Install the preview version of Az.Sql PowerShell package 1.1.1-preview if you are running this PowerShell locally (uncomment below):
+# Install-Module -Name Az.Sql -RequiredVersion 1.1.1-preview -AllowPrerelease -Force
 
 # 1. Create Resource and setup Azure Key Vault (skip if already done)
 
-# Create Resource (name the resource and specify the location)
+# Create Resource group (name the resource and specify the location)
 $location = "westus2" # specify the location
-New-AzResourceGroup -Name "MyRG" -Location $location
+New-AzResourceGroup -Name "MyRG" -Location $location # specify a new RG name
 
-# Create new Azure Key Vault with soft-delete option turned on (change name, RG and region): 
+# Create new Azure Key Vault with a globally unique VaultName and soft-delete option turned on (change VaultName and RG Name): 
 New-AzKeyVault -VaultName "MyKeyVault" -ResourceGroupName "MyRG" -Location $location -EnableSoftDelete
```

Comment:
The comment is now correct, but the vault name should be made a variable. Otherwise users will have to find/replace rather than changing a single value.

Reply:
Done.

![image](https://user-images.githubusercontent.com/40007119/55820993-467fc280-5ab1-11e9-8851-8e34ad5bc9a3.png)